science/imager/Portfile

```diff
@@ -48,7 +48,7 @@ depends_lib-append \
                     path:lib/pkgconfig/harfbuzz.pc:harfbuzz \
                     port:libpng \
                     port:python${py_ver_nodot} \
-                    port:py${py_ver_nodot}-numpy
+                    port:py${py_ver_nodot}-numpy @1.26.4
```

Comment:
that's not an option... you cannot add a version number; revert to the original please